lib/tizen_doctor.dart

```diff
@@ -137,7 +137,7 @@ class TizenValidator extends DoctorValidator {
     }
 
     final Version? sdkVersion = Version.parse(_tizenSdk!.sdkVersion);
-    if (sdkVersion != null && sdkVersion < Version(4, 0, 0)) {
+    if (sdkVersion != null && sdkVersion < Version(5, 0, 0)) {
```

Comment:
just curious. why 5.0? The latest version of tizen studio is 5.5. (Of course, it looks like some more preparations are needed for 5.5. For example.. 8.0 common unified profile support?)

Reply:
Explained in the PR comment :) (although it's very rough)

The minimum requirement for the new feature is 5.0.